integration-tests/main/src/main/java/io/quarkus/it/rest/RestInterface.java

```diff
@@ -18,6 +19,19 @@
     @GET
     String get();
 
+    @GET
+    CompletionStage<String> asyncGet();
+
+    @GET
+    @Path("/jackson")
+    @Produces("application/json")
+    TestResource.MyData getData();
```

Comment:
Is `getData()` called by a test?

Reply:
Hi @kenfinnigan thanks for the review. This is not used anywhere. Good catch. It has been removed now.